ocaml-lsp-server/bin/main.ml

```diff
@@ -1,8 +1,13 @@
 let () =
   Printexc.record_backtrace true;
   let version = ref false in
+  let read_dot_merlin = ref false in
   Arg.parse
-    [ ("--version", Arg.Set version, "print version") ]
+    [ ("--version", Arg.Set version, "print version")
+    ; ( "--read-dot-merlin"
+      , Arg.Set read_dot_merlin
+      , "read Merlin config from .merlin files" )
```

Comment:
should also mention that this needs the `dot-merlin-reader` package installed I believe.